Cargo.toml

```diff
@@ -40,6 +40,7 @@ lazy_static = "1.4.0"
 # Tycho dependencies
 tycho-core = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-core", tag = "0.55.2" }
 tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-client", tag = "0.55.2" }
+tycho-execution = { git = "https://github.com/propeller-heads/tycho-execution.git", package = "tycho-execution", rev = "58749bd65b697673aff94f75954dff5504182041", features = ["evm"] }
```

Comment:
Once we merge [this PR](https://github.com/propeller-heads/tycho-execution/pull/57), I will point this to a version